### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by W4ngatang**
- In `rte-bow` move Winogender to a target task
- Revert breaking `rte-superglue` data loading change
- Return something in Winogender `process_split`
- Change data file from `winogender_filtered.jsonl` -> `Winogender.jsonl` to match server download

### CODE REVIEW

File: jiant/tasks/tasks.py [hunk 1]
```diff
@@ -1416,19 +1416,19 @@ def load_data(self):
         targ_map = {"not_entailment": 0, "entailment": 1}
 
         self.train_data_text = load_pair_nli_jsonl(
-            os.path.join(self.path, "winogender_filtered.jsonl"),
+            os.path.join(self.path, "Winogender.jsonl"),
```
Comment (pruksmhc) [inline jiant/tasks/tasks.py]: I think it should be WInoGender no? (at least how Amanpreet had named the file).

File: jiant/tasks/tasks.py [hunk 2]
```diff
@@ -1545,15 +1548,25 @@ class RTESuperGLUETask(RTETask):
...
-        self.train_data_text = load_pair_nli_jsonl(
-            os.path.join(self.path, "train.jsonl"), self._tokenizer_name, self.max_seq_len, targ_map
-        )
-        self.val_data_text = load_pair_nli_jsonl(
-            os.path.join(self.path, "val.jsonl"), self._tokenizer_name, self.max_seq_len, targ_map
-        )
-        self.test_data_text = load_pair_nli_jsonl(
-            os.path.join(self.path, "test.jsonl"), self._tokenizer_name, self.max_seq_len, targ_map
-        )
+        def _load_jsonl(data_file):
```
Comment (pruksmhc) [inline jiant/tasks/tasks.py]: can you revert this change? This will fail in that winogender_metrics requires pair_id field.

File: jiant/tasks/tasks.py (hunk 1 above)
Comment (W4ngatang) [reply jiant/tasks/tasks.py]: I told him to use `Winogender` (that's how the paper refers to itself)

File: jiant/tasks/tasks.py (hunk 2 above)
Comment (W4ngatang) [reply jiant/tasks/tasks.py]: Why should winogender metrics fail on this RTE example?

File: jiant/tasks/tasks.py (hunk 1 above)
Comment (pruksmhc) [reply jiant/tasks/tasks.py]: Alright - currently it isn't named that way

File: jiant/tasks/tasks.py [hunk 3]
```diff
@@ -1545,15 +1548,25 @@ class RTESuperGLUETask(RTETask):
     def load_data(self):
         """ Process the datasets located at path. """
         targ_map = {"not_entailment": 0, True: 1, False: 0, "entailment": 1}
-        self.train_data_text = load_pair_nli_jsonl(
-            os.path.join(self.path, "train.jsonl"), self._tokenizer_name, self.max_seq_len, targ_map
-        )
```
Comment (pruksmhc) [inline jiant/tasks/tasks.py]: could you change load_pair_nli_json instead of re-inserting it back here? For example, instead of return [sent1s, sent2s, trgs, idxs, pair_ids], perhaps have another condition that only returns the first four items for RTE.

File: jiant/tasks/tasks.py [hunk 4]
```diff
@@ -1545,15 +1548,25 @@ class RTESuperGLUETask(RTETask):
     def load_data(self):
         """ Process the datasets located at path. """
         targ_map = {"not_entailment": 0, True: 1, False: 0, "entailment": 1}
-        self.train_data_text = load_pair_nli_jsonl(
```
Comment (pruksmhc) [inline jiant/tasks/tasks.py]: Also, while you're at it, can you change the superglue-diagnosticheader so that the relative path is RTE/diagnostics?

Comment (W4ngatang) [reply jiant/tasks/tasks.py]: I'll address in the other diagnostics PR

File: jiant/tasks/tasks.py (hunk 3 above)
Comment (W4ngatang) [reply jiant/tasks/tasks.py]: I'd rather not rely on a helper function in another file here. It's pretty straightforward to load a JSONL, and it only gets more complicated with all these different task-specific quirks in a single function.

**Comment by W4ngatang** [comment]
Address #822

File: jiant/tasks/tasks.py (hunk 3 above)
Comment (pruksmhc) [reply jiant/tasks/tasks.py]: Alright - the thinking was basically to reduce the amount of repeated code in the codebase, but up to you!
